### PR TITLE
fix PWM duty cycle setting error, fix #1743

### LINKED ISSUE #1743: [Bug Report]: PWM文档案例中duty cycle设定的值不合理
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
start build

**Comment by skylarCai** [comment]
手动触发！
感谢 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
auto build success,number:201

**Comment by skylarCai** [comment]
autotest fail,number:1537

File: haas_lib_bundles/python/docs/haas_extended_api/driver/pwm.md [hunk 1]
```diff
@@ -135,12 +135,12 @@ print(ret)
 
 * 参数说明：
 设置的参数为字典类型，字典包含 freq 和 duty 两个值。
-如： {'freq':2000, 'duty': 0 }
+如： {'freq':2000, 'duty': 40 }
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/haas_extended_api/driver/pwm.md:138]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Review by wushengtws — APPROVED** [review]

**Pull request merged**